sdk/tests/conformance2/00_test_list.txt

```diff
@@ -4,6 +4,7 @@ context/00_test_list.txt
 extensions/00_test_list.txt
 glsl3/00_test_list.txt
 misc/00_test_list.txt
+offscreencanvas/00_test_list.txt
```

Comment:
Yes, that's most appropriate now, just for bookkeeping purposes. (The 2.0.0 snapshot has already been taken under conformance-suites/ .)